controller/work-item-children-blackbox_test.go

```diff
@@ -46,195 +45,153 @@ type workItemChildSuite struct {
 	spaceCtrl                *SpaceController
 	workItemSvc              *goa.Service
 	typeCtrl                 *WorkitemtypeController
-
 	// These IDs can safely be used by all tests
 	bug1ID               uint64
 	bug2ID               uint64
 	bug3ID               uint64
-	feature1ID           uint64
 	userLinkCategoryID   uuid.UUID
 	bugBlockerLinkTypeID uuid.UUID
 	userSpaceID          uuid.UUID
 
 	// Store IDs of resources that need to be removed at the beginning or end of a test
-	deleteWorkItemLinks []uuid.UUID
-	deleteWorkItems     []string
+	testIdentity account.Identity
+	db           *gormapplication.GormDB
+	clean        func()
 }
 
 // The SetupSuite method will run before the tests in the suite are run.
 // It sets up a database connection for all the tests in this suite without polluting global space.
 func (s *workItemChildSuite) SetupSuite() {
-	var err error
-
-	s.db, err = gorm.Open("postgres", wiConfiguration.GetPostgresConfigString())
-
-	if err != nil {
-		panic("Failed to connect database: " + err.Error())
-	}
-
-	// Make sure the database is populated with the correct types (e.g. bug etc.)
-	if err := models.Transactional(DB, func(tx *gorm.DB) error {
-		return migration.PopulateCommonTypes(context.Background(), tx, workitem.NewWorkItemTypeRepository(tx))
```

Comment:
@kwk Hmm.. then how the tests are passing now? it use data populated by other tests?